semana-16/todolist/src/types.ts

```diff
@@ -0,0 +1,21 @@
+export type User = {
+  id: number;
+  name: string;
+  nickname: string;
+  email: string;
+};
+
+export type Task = {
+  id: number;
+  title: string;
+  description: string;
+  limitDate: string;
+  status: STATUS;
+  user_id: number;
+};
+
+export enum STATUS {
+  TO_DO = "to do",
+  DOING = "doing",
+  DONE = "done"
+}
```

Comment:
Acho interessante também dividir os tipos em arquivos diferentes. Fazer um diretórios "types" e dentro dele os arquivos de cada tipo específico. Acredito que assim,  quem for ler os arquivos do projeto consegue ter um direcionamento melhor de onde tá cada declaração.